src/configs/settings.ts

```diff
@@ -13,6 +13,13 @@
  */
 import { Providers } from '@onaio/gatekeeper';
 import { Expression, LngLatBoundsLike } from 'mapbox-gl';
+import {
+  ActionReasonType,
+  GoalPriorityType,
+  PlanActionCodesType,
+  PlanActivities,
+  UseContextCodesType,
+} from '../containers/forms/PlanForm/types';
```

Comment:
Seems like this is a little circular, but for this type of thing (pun intended) I think it makes sense!

Reply:
yeah, lol, :)